src/controllers/projects.js

```diff
@@ -0,0 +1,67 @@
+import Models from "../../models";
+
+const { Project } = models;
```

Comment:
There is a syntax error here and this read to undefined model. 
```suggestion
const { Project } = Models;
```

Reply:
Nice catch